### PR TITLE
Subset

### PR DESCRIPTION
**Pull request by eti-p-doray**
Implements subset creation.
Datasets are now saved in npz, so files should be regenerated.
neflix_data can now be run directly to generate all necessary files.

### CODE REVIEW

File: netflix_data.py [hunk 1]
```diff
@@ -95,24 +137,33 @@ def split_probe_subset(self, probe_file_path):
...
+        training_size = 0
+        probe_size = 0
+        for i, movie_id in enumerate(self.movie_ids_):
+            user_id = self.user_ids[i]
+            if (movie_id, self.user_map[user_id]) in couples:
+                probe_size += 1
+            else:
+                training_size += 1
+
+        print()
```
Comment (Bettorun) [inline netflix_data.py]: Needed?

File: netflix_utils.py [hunk 2]
```diff
@@ -87,9 +87,9 @@ def parse_args(self, argv):
         parser = argparse.ArgumentParser(description="""
         Simple matrix factorization algorithm to find latent variables that represents netflix prize data.
         """, formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-        parser.add_argument('-i', '--input', default = 'nf_prize_dataset/nf_probe_training.npy',
+        parser.add_argument('-i', '--input', default = 'nf_prize_dataset/nf_training.npz',
```
Comment (Bettorun) [inline netflix_utils.py:90]: Just to be sure, we can load a npz the same way we previously loaded npy?

**Review by Bettorun — APPROVED** [review]

File: netflix_data.py (hunk 1 above)
Comment (eti-p-doray) [reply netflix_data.py]: Whoops

File: netflix_utils.py (hunk 2 above)
Comment (eti-p-doray) [reply netflix_utils.py:90]: Yep, au final ça marche pareil (faut juste regénérer les fichiers)

**Comment by Bettorun** [comment]
LGTM

**Pull request merged**